src/content/1.7/basics/keeping_up-to-date/upgrade.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Upgrade
+weight: 30
+---
+
+# How-to: Upgrade
+
+
+{{% notice warning %}}
+**Important**
+
+Do not go further if you haven’t made a backup of your shop.
```

Comment:
How about a link to the related article?

```markdown
Learn [how to backup your shop]({{< ref "/1.7/basics/keeping_up-to-date/preliminary_task.md" >}})
```

Reply:
Done!